src/spdx2/ui/SpdxTwoGeneratorUi.php

```diff
@@ -65,6 +65,9 @@ function preInstall()
     $text = _("Generate SPDX report in tag:value format");
     menu_insert("Browse-Pfile::Export&nbsp;SPDX&nbsp;tag:value", 0, self::NAME . '&outputFormat=spdx2tv', $text);
 
+    $text = _("Generate SPDX report in CSV format");
+    menu_insert("Browse-Pfile::Export&nbsp;CSV&nbsp;report", 0, self::NAME . '&outputFormat=spdx2csv', $text);
```

Comment:
Can we call it "Export SPDX CSV report" as there is already a CSV report under "Export List" in a package. It can be confusing.

![image](https://user-images.githubusercontent.com/18077542/147914659-8167ad82-1faa-4c04-ae3c-5155041cd6a7.png)

Reply:
I didn't go for "SPDX CSV" naming in the first place since it is not a ["standard" SPDX format](https://spdx.dev/resources/use/) - I feared that it would be confusing too :

> SPDX Documents can use one of five file formats, tag/value (.spdx), JSON(.spdx.json), YAML(.spdx.yml), RDF/xml(spdx.rdf) and spreadsheets (.xls).

Maybe, to improve over clarity, we should rename all "Reports" naming ?
![image](https://user-images.githubusercontent.com/22956329/147918491-1148e7a9-0e1c-4cbf-9b17-02fcd1392282.png)

To have something like:
- ...
- Export Report: CSV
- Export Report: DEP5
- Export Report: ReadMe_OSS
- Export Report: SPDX RDF
- Export Report: DPDX tag:value
- Export Report: Unified Report
- Import Report
- ...

Or
- ...
- Export report to CSV
- Export report to DEP5
- Export report to ReadMe_OSS
- Export report to SPDX RDF
- Export report to SPDX tag:value
- Export report to Unified Report
- Import Report
- ...

?

Reply:
I am not sure what will be the best option here as there is already a CSV report in FOSSology with different functionality.

How about "Export report to SPDX_ID CSV" (and move it closer to SPDX reports) as one of the difference between the two formats is the additional `SPDXID` field?

But I agree about renaming the export statements to "Export report to SPDX RDF" format.

Reply:
Indeed, I agree the two CSV export can be confusing :-/

Actually, the SPDXID is not mandatory for our use. We can remove it to unlink it from SPDX ties.

The differences between the existing "Export List" and the new "export to CSV" is that the latter include the LicenseComments field, and aggregate Licensing and Copyright data.

The way I see it, the existing "Export List" is more for studying the results with precision (with many options), and the new one generates a more complete, "report like" CSV export, to communicate a lighter-than-actual SPDX report standard for human review.

Maybe renaming all actions like this would emphasize the difference between the `CSV report ` and  `Export List` features ?
-     ...
-     Export CSV report
-     Export DEP5 report
-     Export ReadMe_OSS report
-     Export SPDX RDF report
-     Export PDX tag:value report
-     Export Unified report
-     Import Report
-     ...

Reply:
may be something like "Export CSV report(SPDX)"

Reply:
Just wanted to confirm before approving @NicolasToussaint , you want to name the option as "Export SPDX CSV report (SPDX)"?

![image](https://user-images.githubusercontent.com/18077542/148045909-5b6ff0db-67b0-4db2-902d-8cc87ad7ec7d.png)

Reply:
> Just wanted to confirm before approving @NicolasToussaint , you want to name the option as "Export SPDX CSV report (SPDX)"?

Arg, I really need some sleep ... I meant `Export CSV report (SPDX)` as proposed